src/Node/AbstractNode.php

```diff
@@ -80,10 +80,14 @@ public function getId(): string
     public function getPath(): string
     {
         if ($this->path === null) {
-            if ($this->parent === null || $this->parent->getPath() === null || $this->parent->getPath() === false) {
+            if (
+                $this->parent === null ||
```

Comment:
Please do not mix reformatting like this with what the pull request is actually about.